src/library/module_mgr.cpp

```diff
@@ -177,25 +180,28 @@ void module_mgr::build_module(module_id const & id, bool can_use_olean, name_set
             }
 
             mod->m_lt = lt.get();
-            mod->m_trans_mtime = mod->m_mtime;
+            unsigned trans_hash = mod->m_src_hash;
```

Comment:
This would change the semantics, since `trans_hash` here is the *actual* value which is later compared to `mod->m_trans_hash`. I changed its name to reflect this.